packages/styles/src/components/form-validation.scss

```
@@ -18,7 +18,7 @@
 
 .invalid-feedback,
 .valid-feedback {
-  display: none;
+  display: flex;
```

This change will show validation feedback in certain situations when it's rendered in the DOM but the control has not been validated yet. See line https://github.com/swisspost/design-system/pull/3396/files#diff-1c6c2dabdd404cc629b0659cde023035d93529fe69b3798056ef1b04ef8dc8c4L92 where the control gets `display: inline-flex` if there's a sibling with `is-valid` or `is-invalid` only.

```suggestion
  display: none;
```